app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:create], if: -> { request.format.json? }
+
+  def index
+    post = Post.find(params['post_id'])
+    comments = post.comments
+    render json: comments
+  rescue ActiveRecord::RecordNotFound
+    render json: { error: 'Post not found' }, status: :not_found
+  end
+
+  def create
+    user_id = params['user_id']
+    post = Post.find(params['post_id'])
+
+    user = User.find(user_id)
+
+    if user.nil?
+      render json: { error: 'User not found' }, status: :not_found
+      return
+    end
+
+    new_comment = post.comments.new(
+      text: params['text'],
+      author: user,
+      post:
+    )
+
```

Comment:
- [ ] It would be nice if you attach a screenshot of the email in the pull request summary which you get from microverse allowing you to work alone on this milestone 👍 so the next reviewer can verify it and proceed review on your pull request 😊